x/conflict/README.md

```diff
@@ -0,0 +1,124 @@
+
+### Commit Period
+
+This is the voting period of the conflict, providers that are selected as jury are required to submit their hashed vote.
+The hash consisnt of nonce + response hash + provider address
+
+```go
+type MsgConflictVoteCommit struct {
+	Creator string // provider address
+	VoteID  string // the conflict unique id
+	Hash    []byte // the hashed nonce + response + provider address
+}
+```
+
+### Reveal Period
+
+Once the commit period ends, the conflict enters reveal state. Here providers need to submit their full response and nonce. this is than validated and compared to the original providers responses. The result of the vote (Provider A, Provider B, None) is than saved on state, waiting for the end of the conflict.
+
+```go
+type MsgConflictVoteReveal struct {
+	Creator string // provider address
+	VoteID  string // the conflict unique id
+	Nonce   int64  // random nonce
+	Hash    []byte // hash of the response
+}
+```
+
+### Conflict Resolve
+
+Once the reveal period ends votes are than counted to determine the results of the conflict. Votes are weighted by providers stake. 
+Providers in the jury that did not vote are punished by jail and slashing, their slashed amount will be added to the conflict reward pool.
+For the conflict resolution there needs to be a majority met of votes for Provider A, Provider B or None of them. 
+If a majority was not met, conflict reward pool is given to the consumer that reported the conflict.
+Once a majority is met providers that voted to the wrong side of the conflict are slashed and frozen, the slashed amount is added to the conflict reward pool.
+Now the reward pool is distributed between the comsumer and the providers that voted for the correct provider.
+
+## Parameters
+
+The conflict module contains the following parameters:
+
+| Key                                    | Type                    | Default Value    |
+| -------------------------------------- | ----------------------- | -----------------|
+| MajorityPercent                        | math.LegacyDec          | 0.95              |
+| VoteStartSpan                              | uint64          | 3             |
+| VotePeriod                       | uint64          | 2                |
+| Rewards                        | rewards                  | N/A                |
+
+`MajorityPercent` majority needed to conclude who is the winner of the conclict
+`VoteStartSpan` is the amount of epochs in the past that a consumer can send conflict detection message for
+`VotePeriod` is the amount of epochs for each voting state (commit/reveal)
+`Rewards` defines how the reward pool tokens will be distributed between all participants
+
+```go
+type Rewards struct {
+	WinnerRewardPercent github_com_cosmos_cosmos_sdk_types.Dec // the winnig provider of the conflict portion
```

Comment:
-> the conflict's winner portion (provider)

Reply:
done